Sprint-2/5-stretch-extend/format-time.js

```diff
@@ -3,23 +3,34 @@
 // Your task is to write tests for as many different groups of input data or edge cases as you can, and fix any bugs you find.
 
 function formatAs12HourClock(time) {
-  const hours = Number(time.slice(0, 2));
-  if (hours > 12) {
-    return `${hours - 12}:00 pm`;
-  }
-  return `${time} am`;
+  let hours = Number(time.slice(0, 2));
+  const minutes = time.slice(3, 5);
+  const ampm = hours >= 12 ? `pm` : `am`;
+
+  if (hours === 0) {
+    hours = 12;
+  } else if (hours > 12) hours = hours - 12;
```

Comment:
I'd recommend always using `{}`s around the body of `if`/`else` statements. As well as consistency (which is good), you can read more about some of the problems not using `{}`s causes at https://www.blackduck.com/blog/understanding-apple-goto-fail-vulnerability-2.html